sensor/common/detector/metrics/metrics.go

```diff
@@ -88,6 +88,38 @@ var (
 			"ack_type",
 			"reason",
 		})
+
+	// DetectorProcessIndicatorBufferSize keeps track of the size of the detection process indicator buffer.
+	DetectorProcessIndicatorBufferSize = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: metrics.PrometheusNamespace,
+		Subsystem: metrics.SensorSubsystem.String(),
+		Name:      "detector_process_indicator_buffer_size",
+		Help:      "A counter that tracks the size of the detection process indicator buffer",
+	}, []string{"Operation"})
+
+	// DetectorNetworkFlowBufferSize keeps track of the size of the detection network flow buffer.
+	DetectorNetworkFlowBufferSize = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: metrics.PrometheusNamespace,
+		Subsystem: metrics.SensorSubsystem.String(),
+		Name:      "detector_network_flow_buffer_size",
+		Help:      "A counter that tracks the size of the detection network flow buffer",
+	}, []string{"Operation"})
+
+	// DetectorProcessIndicatorDroppedCount keeps track of the number of process indicators dropped in the detector.
+	DetectorProcessIndicatorDroppedCount = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: metrics.PrometheusNamespace,
+		Subsystem: metrics.SensorSubsystem.String(),
+		Name:      "detector_process_indicator_dropper_counter",
```

Comment:
According to https://prometheus.io/docs/practices/naming/#metric-names, we should rather name it 
```suggestion
		Name:      "detector_process_indicators_dropped_total",
```

Reply:
Done